src/android_components.py

```diff
@@ -166,6 +192,25 @@ def _update_geckoview_new(
             f"{ts()} Latest GV {gv_channel.capitalize()} version available is {latest_gv_version}"
         )
 
+        # TODO GeckoView 94 and above provide reliable Glean support.
+        if int(current_gv_major_version) >= 94:
```

Comment:
We won't need to handle < 94 with this anymore now, so I guess we can remove this?

Reply:
Yeah if versions older than 94 don't ship anymore then we can get rid of this conditional.